css/style.css

```diff
@@ -0,0 +1,29 @@
+*{
```

Comment:
I would change this to the `body` tag and have these styles inherit down the rest of the document.